roles/mongodb/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+# MongoDB version to install
+# Should be '3.2' or '3.4'
+mongodb_version: "3.4"
```

Comment:
@cognifloyd this looks like exactly what you asked recently.

Reply:
Yeah. This looks very nice. It's pretty much what I did, though I still used the key directly from mongo.
Hopefully I'll have time soon to push PRs for everything else I fixed (though I don't need to push my mongodb role changes now.) :+1: